src/index.js

```diff
@@ -94,6 +94,15 @@ window.App = {
     const myNametag = document.querySelector("#player-rig .nametag");
     myNametag.setAttribute("text", "value", username);
 
+    //change spawn position & rotation
+    const myHead = document.querySelector("#player-rig #head");
+    let rotation = Math.random() * 2 * Math.PI;
+    let position = myHead.getAttribute("position");
+    let newPosition = getSpawnPositionInCircle(0, 0, 4, rotation);
+    newPosition.y = position.y;
+    myHead.setAttribute("position", newPosition);
+    myHead.setAttribute("rotation", getRotationToTarget(newPosition, {x:0, y:0, z:0}));
```

Comment:
Object3D has a lookAt funciton, I would just use that https://threejs.org/docs/#api/core/Object3D, though you will still have to call setAttribute to update the aframe side of things like we do here Threejs has a constant for this `THREE.Math.RAD2DEG` so I would ditch this functin and just do the multiplication inline like we do here https://github.com/mozilla/mr-social-client/blob/b08841cdaaa800b0ff96b1f5ccd3b453314f886d/src/components/character-controller.js#L257-L261 (its a bit silly because you end up doing RAD -> DEG -> RAD, but that's just a result of aframe deciding to use degrees instead of radians.